Sources/AsyncHTTPClient/ConnectionPool/RequestBodyLength.swift

```diff
@@ -12,9 +12,16 @@
 //
 //===----------------------------------------------------------------------===//
 
-enum RequestBodyLength: Hashable {
+public struct RequestBodyLength {
```

Comment:
We should not add this type without prefix to the top namespace.

Reply:
Agreed. I have reverted the change and added a new public type `HTTPClientRequest.Body.Length` which is baked by `RequestBodyLength` and used as the public type instead.